integration/flows/popover.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '@playwright/test';
```

Comment:
Rather than "flows" how about "tests". I expect this will contain tests for many components, behaviors, etc.